lib/protobuf/builder.ex

```diff
@@ -28,7 +28,7 @@ defmodule Protobuf.Builder do
       # strict.
       %{__struct__: _other_mod} when strict? ->
         raise ArgumentError,
-          message: "The __struct__ in the struct doesn't with the message module"
+          message: "The module in __struct__ doesn't match the message module"
```

Comment:
Unrelated: we could probably improve this error message by detailing the mismatch.